src/compose/ComposeMenu.js

```diff
@@ -140,28 +142,54 @@ export default function ComposeMenu(props: Props): Node {
         return;
       }
 
-      // TODO: support sending multiple files; see library's docs for how to
-      // let `assets` have more than one item in `response`.
-      const firstAsset = response.assets && response.assets[0];
-
-      const { uri, fileName } = firstAsset ?? {};
+      const { assets } = response;
 
-      if (!firstAsset || uri == null || fileName == null) {
+      if (!assets || !assets[0]) {
         // TODO: See if we these unexpected situations actually happen. …Ah,
         //   yep, reportedly (and we've seen in Sentry):
         //   https://github.com/react-native-image-picker/react-native-image-picker/issues/1945
-        showErrorAlert(_('Error'), _('Something went wrong, and your message was not sent.'));
-        logging.error('Unexpected response from image picker', {
-          '!firstAsset': !firstAsset,
-          'uri == null': uri == null,
-          'fileName == null': fileName == null,
+        showErrorAlert(_('Error'), _('Failed to attach your file.'));
+        logging.error('Image picker response gave falsy `assets` or falsy `assets[0]`', {
+          '!assets': !assets,
         });
         return;
       }
 
-      dispatch(uploadFile(destinationNarrow, uri, chooseUploadImageFilename(uri, fileName)));
```

Comment:
Looks like this was the only call site of that `uploadFile` function — so let's cut that out too.